package.json

```diff
@@ -5,9 +5,24 @@
     "postinstall": "lerna bootstrap --hoist",
     "updateApis": "lerna run updateApis",
     "build": "lerna run build",
+    "redis": "npm run redis --prefix ./package/core",
+    "redis:shutdown": "npm run redis:shutdown --prefix ./package/core",
+    "redis:debug": "npm run redis:debug --prefix ./package/core",
+    "redis:debug:shutdown": "npm run redis:debug:shutdown --prefix ./package/core",
     "test": "lerna run test",
     "test:debug": "lerna run test:debug",
-    "test:integration": "lerna run test:integration",
+    "test:integration:bundle": "lerna run test:integration",
+    "test:integration:bundle:redis": "lerna run test:integration:redis",
+    "test:integration:bundle:redis:debug": "lerna run test:integration:redis:debug",
+    "test:integration": "npm run test:integration:bundle --prefix ./integrationTests",
+    "test:integration:redis": "npm run test:integration:redis --prefix ./integrationTests",
+    "run test:integration:redis:debug": "npm run test:integration:redis:debug --prefix ./integrationTests",
+    "test:all": "npm run test && npm run test:integration",
+    "test:all:redis": "npm run test && npm run test:integration:bundle:redis",
```

Comment:
Add --prefix integrationTests

Reply:
no, this is an internal call to the script on line 15